finesse/gui/temp_control.py

```diff
@@ -71,15 +66,15 @@ def _create_controls(self) -> QGridLayout:
         self._btns["cold"].clicked.connect(
             partial(self._toggle_axis_visibility, name="cold")
         )
+        self._btns["hot"].setMaximumWidth(80)
+        self._btns["cold"].setMaximumWidth(80)
+
         self._create_figure()
         self._canvas.setMinimumSize(QSize(640, 120))
-        self._canvas.setSizePolicy(
-            QSizePolicy.Policy.MinimumExpanding, QSizePolicy.Policy.MinimumExpanding
-        )
 
         layout.addWidget(self._btns["hot"], 0, 0)
         layout.addWidget(self._btns["cold"], 1, 0)
-        layout.addWidget(self._canvas, 0, 1, 2, 1)
+        layout.addWidget(self._canvas, 0, 1, 3, 1)
```

Comment:
I never would have thought to try that 😅. I think this makes it look much nicer with the buttons in the top corner as well.